### PR TITLE
Simple user login system

### PR DESCRIPTION
**Pull request by Brianktoo**
add 'configuration handling' in simple user login system to handle configurations

### CODE REVIEW

File: simple-user-login-system/run.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+
+
+
+app = create_app()
+app.register_blueprint(user_bp)
+
+env=os.getenv("ENV")
+print(f'you are working on {env} environment')
+print(app.config['DB'])
```
Comment (codetaskmaster) [inline simple-user-login-system/run.py:17]: Please remove these print calls

File: simple-user-login-system/run.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+print(app.config['PASSWORD'])
+print(app.config['USER'])
+
+
+
+if __name__ == "__main__":
+    app.run(debug=True)
+
```
Comment (codetaskmaster) [inline simple-user-login-system/run.py:25]: Add white space

File: simple-user-login-system/app/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+        # Compare database password with given password
+        if record and record[1] == password:
+            return True
+
+        # Close cursor
+        self.db.cursor.close()
+        # Close connection
+        self.db.conn.close()
+
```
Comment (codetaskmaster) [inline simple-user-login-system/app/models/__init__.py:44]: Add white space

File: simple-user-login-system/app/db/__init__.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        self.password = app.config['PASSWORD']
+
+        # Create a db connection
+        self.conn = mysql.connector.connect(
+            host=self.host, database=self.db, user=self.user, password=self.password
+        )
+
+        # Create cursor
+        self.cursor = self.conn.cursor()
+
```
Comment (codetaskmaster) [inline simple-user-login-system/app/db/__init__.py:24]: Add white space

**Review by codetaskmaster — CHANGES_REQUESTED** [review]

**Comment by codetaskmaster** [comment]
Also remove all the `__pycache__` files in your remote repo

File: simple-user-login-system/app/__init__.py [hunk 5]
```diff
@@ -0,0 +1,14 @@
+from flask import Flask
+import os
+
+
+def create_app():
+    app = Flask(__name__)
+    if os.getenv("ENV")=="development":
+        app.config.from_object("config.DevelopmentConfig")
```
Comment (codetaskmaster) [inline simple-user-login-system/app/__init__.py:8]: Where does the `config.DevelopmentConfig` object come from?

File: simple-user-login-system/run.py (hunk 1 above)
Comment (Brianktoo) [reply simple-user-login-system/run.py:17]: hoo! sure, i forget to remove it i was actually using it to check if my application working environment is changing when i switch it in .env. let me remove it.

File: simple-user-login-system/app/__init__.py (hunk 5 above)
Comment (Brianktoo) [reply simple-user-login-system/app/__init__.py:8]: my config.DevelopmentConfig object is comming from my config.py file which i have acctually added it to git ignore.

Comment (Brianktoo) [reply simple-user-login-system/app/__init__.py:8]: This is actually how my configuration looks like in config.py file

class Config(object):

    DEBUG = False
    TESTING = False
    SECRET_KEY = ''
    HOST = 'localhost'
    DB = 'production_db'
    USER = 'admin'
    PASSWORD = 'password'
    UPLOADS = ''
    SESSION_COOKIE_SECURE = True

class ProductionConfig(Config):
    pass

class DevelopmentConfig(Config):
    DEBUG = True    
    HOST = "localhost"
    DB = "login"
    USER = "root"
    PASSWORD = "Tyondo@98" 
    SESSION_COOKIE_SECURE = False

class TestingConfig(Config):
    TESTING = True  
    HOST = "localhost"
    DB = "logintest"
    USER = "root"
    PASSWORD = 'Tyondo@98'
    SESSION_COOKIE_SECURE = False

Comment (Brianktoo) [reply simple-user-login-system/app/__init__.py:8]: so nikiwa kwa development environment natumia login database, nikiswitch to testing environment natumia logintest database.

**Comment by codetaskmaster** [comment]
> Also remove all the `__pycache__` files in your remote repo

@Brianktoo

File: simple-user-login-system/app/__init__.py (hunk 5 above)
Comment (Brianktoo) [reply simple-user-login-system/app/__init__.py:8]: Done.

File: simple-user-login-system/config.py [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    PASSWORD = os.getenv('PASSWORD')
+
+class TestingConfig(Config):
+    TESTING = '' 
+    HOST = ''
+    DB = ''
+    USER = ''
+    PASSWORD = ''
+    
+
```
Comment (codetaskmaster) [inline simple-user-login-system/config.py:29]: White line

File: simple-user-login-system/templates/styles.css [hunk 7]
```diff
@@ -0,0 +1,261 @@
...
+        width: 220px;
+        font-size: 15px;
+    }
+
+    .clkbtn {
+        height: 50px;
+        width: 130px;
+        font-size: 19px;
+    }
+}
```
Comment (codetaskmaster) [inline simple-user-login-system/templates/styles.css:261]: white line

File: simple-user-login-system/app/db/__pycache__/__init__.cpython-310.pyc
Comment (codetaskmaster) [inline simple-user-login-system/app/db/__pycache__/__init__.cpython-310.pyc:1]: Delete these

**Review by codetaskmaster — CHANGES_REQUESTED** [review]

File: simple-user-login-system/app/db/__pycache__/__init__.cpython-310.pyc
Comment (Brianktoo) [reply simple-user-login-system/app/db/__pycache__/__init__.cpython-310.pyc:1]: git rm -f *.pyc

**Review by codetaskmaster — APPROVED** [review]
Great job!

**Comment by Brianktoo** [comment]
done

**Pull request merged**